.github/workflows/cron-streams-active-cleanup.yaml

```diff
@@ -0,0 +1,15 @@
+name: "cron: Active streams cleanup"
+
+on:
+  schedule:
+    - cron: "*/10 * * * *" # run every 10 minutes
```

Comment:
How about running it every 5 min and getting rid of all the opportunistic cleanup? I think this job should be the only place where we do the stream cleanup.

Otherwise, we risk not exposing a lot of bugs + it's super wrong that making a GET call to studio makes a change in the Studio DB.

Reply:
Super scary, but I also would love to remove the opportunistic logic. Implemented here! https://github.com/livepeer/studio/pull/2013/commits/f70dfe3b4ff708d21eb97a596db35e45ba03013a

Notice that with a 5 minute frequency, the worst case delay for a stream to be activecleanedup will be 10 minutes. I think making the github action more frequent would not be that good though given limited runners etc, so we can consider that once we move to an actual cron in our infra. WDYT?